summarization/stamp_page_picking/interesting_sequence_picker.py

```diff
@@ -62,7 +62,7 @@ def __init__(
         self._set_seed_stamp_page()
 
     def get_interesting_sequence_and_unused_pages(self):
-        for self.iteration_count in range(self.max_pages_allowed):
+        for self.iteration_count in range(self.max_pages_allowed-1):
```

Comment:
We are reducing the number of iterations here because `self.stamp_page_sequence` already has a seed (title) page right? But, since we always add a title page to the final STAMP document, would it make more sense to remove the seed page from `self.stamp_page_sequence` instead?

Reply:
updated!